build-operator.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# script to make kudo operators
+# takes form of `./build-operator.sh  repository/kafka/operator/`
+# or `./build-operator kafka` which assumes the above common layout
+# or `./build-operator cassandra 3.11`
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CMD=kubectl-kudo
+REPO_DIR=build/repo
+
+
+# script requires kudo `kubectl-kudo` to be installed and in the path
+command -v $CMD >/dev/null 2>&1 || { echo >&2 "$CMD is required in the path.  Aborting."; exit 1; }
+
+# which operator to build must be passed
+if [ $# -lt 1 ]; then
+  echo 1>&2 "Usage: $0 path_to_operator [optional_version]"
+  exit 1
+fi
+
+# convenience assumption which allow for operator name only
+# for pass in.  `./build-operator kafka`
```

Comment:
this might give the idea that `./build-operator cassandra` would build the operator also when `operator.yaml` is residing inside `cassandra/3.11/operator/`
why not ask for the path to `operator.yaml` directory?
```
./build-operator cassandra/3.11/operator
./build-operator kafka/operator
```

Reply:
while I understand... at that point I would rather the full path be provided which already works.   The other options are for convenience... but if they are confusing perhaps we should remove them.